src/_locales/es/messages.json

```diff
@@ -166,23 +166,23 @@
         "description": "Checkbox label on the general settings page"
     },
     "show_counter_checkbox": {
-        "message": "Mostrar recuento de elementos bloqueados",
+        "message": "Mostrar el número de elementos bloqueados",
         "description": "Checkbox label on the general settings page"
     },
     "what_is_a_tracker": {
         "message": "¿Qué es un rastreador?",
         "description": "Tooltip that comes up when you hover over the 'tracking domains' link under the Tracking Domains tab on the options page."
     },
     "intro_report_button": {
-        "message": "Por favor no olvides hacer clic en 'Reportar un sitio roto'. Respetamos tu privacidad por lo que no enviamos reportes de forma automática.",
+        "message": "Por favor, no olvide hacer clic en «Reportar un sitio roto». Respetamos su privacidad, por lo que no enviamos reportes de forma automática.",
```

Comment:
Need to update to match the updated text in `report_broken_site`. Just the first part should be enough: "clic en «¿Ha roto Privacy Badger este sitio?»"

Reply:
Done.